internal/signing/store/channel_accounts_model.go

```diff
@@ -19,23 +19,6 @@ var (
 	ErrChannelAccountNotFound        = errors.New("channel account not found")
 )
 
```

Comment:
How about we just move the model to the `data` package alongside the other models?

Reply:
Here it's a different concept from the regular models that we've on the `data` package. This is the default implementation of the `Store` interface. This coincidentally uses the database, but that's not a rule, this could be stored anywhere, and that's why we have this model separated from the others.

Reply:
Makes sense 👍